_IBC/canto-carbon.json

```diff
@@ -2,30 +2,29 @@
   "$schema": "../ibc_data.schema.json",
   "chain_1": {
     "chain_name": "canto",
-    "client_id": "07-tendermint-10",
-    "connection_id": "connection-10"
+    "client_id": "07-tendermint-47",
+    "connection_id": "connection-37"
   },
   "chain_2": {
     "chain_name": "carbon",
-    "client_id": "07-tendermint-40",
-    "connection_id": "connection-23"
+    "client_id": "07-tendermint-63",
+    "connection_id": "connection-44"
   },
   "channels": [
     {
       "chain_1": {
-        "channel_id": "channel-6",
+        "channel_id": "channel-29",
```

Comment:
We generally cannot change the IBC channel for any registered connections (carbon to: canto, osmosis, stargaze). We'll need proof of either:
- governance on each chain indicating agreement to change the IBC channels, or
- proof that there are not abandoned tokens on counterparty chains over the channels being removed

Reply:
@JeremyParish69 Apologies, I've reverted the changes for canto and stargaze.

As for osmosis, I only changed some indentation: the IBC channel information is the same. Could you review the changes again? Thank you!